MTVd-test-project/css/about.css

```diff
@@ -0,0 +1,74 @@
+.about-content-box {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: -webkit-calc(100% - 200px);
```

Comment:
Do not use browser-specific declarations, like -webkit-calc. Use just calc instead.
Check for same usages over the whole project.